Cython/Utility/Exceptions.c

```diff
@@ -482,17 +501,52 @@ static CYTHON_INLINE void __Pyx__ExceptionReset(PyThreadState *tstate, PyObject
 #define __Pyx_ExceptionReset(type, value, tb)  PyErr_SetExcInfo(type, value, tb)
 #endif
 
+/////////////// _ExceptionGetType.proto ///////////////
+
+static CYTHON_INLINE PyObject* __Pyx__ExceptionGetType(PyObject *value) {
+    if (value != NULL && value != Py_None) {
+        return (PyObject*)Py_TYPE(value);
+    }
+    else {
+        return NULL;
+    }
+}
+
+
+/////////////// _ExceptionGetTraceback.proto ///////////////
+
+// Used by __Pyx__ExceptionSave() and __Pyx__ExceptionSwap() on Python 3.11a6
+static CYTHON_INLINE PyObject* __Pyx__ExceptionGetTraceback(PyObject *value) {
```

Comment:
And this needs to be defined on Python 3.4+ only. On C++ the lack of declaration for `PyException_GetTraceback` is causing it to fail